config/locales/en.yml

```diff
@@ -294,7 +294,9 @@ en:
     otp_code: OTP code
     require_mfa_disabled: Your multi-factor authentication has been enabled. To reconfigure multi-factor authentication, you'll have to disable it first.
     require_mfa_enabled: Your multi-factor authentication has not been enabled. You have to enable it first.
+    setup_required_html: For protection of your account and your gems, you are required to set up multi-factor authentication. See the <a href="https://blog.rubygems.org/2022/08/15/requiring-mfa-on-popular-gems.html">blog post</a> for more information.
```

Comment:
```suggestion
    setup_required_html: For protection of your account and your gems, you are required to set up multi-factor authentication. Please read our <a href="https://blog.rubygems.org/2022/08/15/requiring-mfa-on-popular-gems.html">blog post</a> for more details.
```
nit suggestion on the phrasing of the blog post

Also, I noticed the default style on the link would violate the colour palette of RubyGems. Could we add a style for links in a flash div to not be blue? Maybe underline would be more appropriate.

<img width="1079" alt="Screen Shot 2022-08-10 at 1 27 13 PM" src="https://user-images.githubusercontent.com/42748004/183977599-281b9fbe-accd-489c-8e81-c64d4b36f227.png">